pkg/devfile/parser/data/2.1.0/devfileJsonSchema210.go

```diff
@@ -1198,7 +1198,44 @@ const JsonSchema210 = `{
               "source"
             ],
             "type":"object"
-          }
+          },
+          "SourceToImage":{
+            "description":"SourceToImage component",
+            "properties":{
+              "name":{
+                "description":"Mandatory name that allows referencing the sourcetoimage component",
+                "type":"string"
+              },              
+              "fromKind":{
+                "description":"From Kind",
+                "type":"string"
+              },
+              "fromNamespace":{
+                "description":"From Namespace",
+                "type":"string"
+              },
+              "fromName":{
+                "description":"From Name",
+                "type":"string"
+              },
+              "script": {
+                "description":"Script URL path to override default scripts provided by builder image",
+                "type":"string"
+              },
+              "incrementalbuild":{
+                "description":"Flag to perform increamental builds",
+                "type": "boolean"
+              }
+                         
+            },
+            "required":[
+              "name",
+              "fromKind",
+              "fromNamespace",
+              "fromName"
```

Comment:
Not sure I understand what these fields represent and why they need to be mandatory 
do these refer to the builder image name & namespace?